src/app/controllers/OverviewController.ts

```diff
@@ -148,11 +149,22 @@ export default class OverviewController {
 		}
 
 		let attrEffectiveness = 0
+		let displayAttrEffectiveness = false
 		if (activeValidators.length > 0) {
-			attrEffectiveness = sumBigInt(validators, (cur) => (cur.attrEffectiveness ? new BigNumber(cur.attrEffectiveness.toString()) : new BigNumber(0)))
+			displayAttrEffectiveness = true
+
+			attrEffectiveness = sumBigInt(activeValidators, (cur) =>
+				cur.attrEffectiveness ? new BigNumber(cur.attrEffectiveness.toString()) : new BigNumber(0)
+			)
 				.dividedBy(activeValidators.length)
 				.decimalPlaces(1)
 				.toNumber()
+
+			const attrChecked = Math.min(Math.max(attrEffectiveness, 0), 100)
```

Comment:
Because of the change above, this should be obsolete. But better save than sorry, I guess.